cdisc_rules_engine/dataset_builders/variables_metadata_with_define_dataset_builder.py

```diff
@@ -28,13 +29,22 @@ def build(self):
         define_variable_has_codelist,
         define_variable_codelist_coded_values
         """
+        validate_xml = kwargs.get("validate_xml", "Y")
+        logger = logging.getLogger()
```

Comment:
We should use the logger from: https://github.com/cdisc-org/cdisc-rules-engine/blob/main/cdisc_rules_engine/services/logging/logging_service_factory.py#L35